.github/workflows/push-bundles.yaml

```diff
@@ -18,6 +16,10 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@v3
+      with:
+        # So we can see in which commit a bundle's content was
+        # most recently updated
+        fetch-depth: 0
```

Comment:
Something like 100 would probably work I guess...

Reply:
I think it's okay, should be local to GitHub Actions and the repository is not that big...